test/test_nameko_socket_server.py

```diff
@@ -23,9 +23,9 @@ def test_server(container_factory):
     container.start()
 
     sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    sock.connect(('', 6000))
-    sock.sendall('hello\n')
-    data = sock.recv(1024)
+    sock.connect(('127.0.0.1', 6000))
+    sock.sendall('hello\n'.encode('UTF-8'))
+    data = sock.recv(1024).decode('UTF-8')
```

Comment:
sorry, to explain what i meant: 

```
sock.sendall(b'hello\n')
# ...
assert data == b'hello world'
```

but as you say, encoding might make for a nicer example. personally i think `.encode('utf-8')` (lowercase) reads a little nicer but up to you